explorer/app/components/Loading/loading.tsx

```diff
@@ -6,18 +6,19 @@
 import { Typography } from "@mui/material";
 import React, { useEffect, useRef } from "react";
 import { LoadingIcon } from "../common/CustomIcon/components/LoadingIcon/loadingIcon";
+import { PaperPanelStyle, PAPER_PANEL_STYLE } from "../common/Paper/paper";
 import { LoadingBackground } from "./loading.styles";
 
 interface Props {
-  className?: string;
   loading: boolean;
+  panelStyle?: PaperPanelStyle; // Enables loading to mirror parent container styles.
   text?: string;
 }
 
 export const Loading = ({
-  className,
   loading,
-  text = "Your link will be ready shortly...",
```

Comment:
Removed the assignment of "Your link..." when text is undefined for the benefit of using the Loading component on the Index table.